plugins/inputs/azure_storage_queue/azure_storage_queue.go

```diff
@@ -99,19 +99,19 @@ func (a *AzureStorageQueue) Gather(acc telegraf.Accumulator) error {
 		marker = queuesSegment.NextMarker
 
 		for _, queueItem := range queuesSegment.QueueItems {
-			a.Log.Debugf("Processing queue '%s' of storage account '%s'", queueItem.Name, a.StorageAccountName)
+			a.Log.Debugf("Processing queue %q of storage account %q", queueItem.Name, a.StorageAccountName)
 			queueURL := serviceURL.NewQueueURL(queueItem.Name)
 			properties, err := queueURL.GetProperties(ctx)
 			if err != nil {
-				a.Log.Errorf("Error getting properties for queue %s: %s", queueItem.Name, err.Error())
+				a.Log.Errorf("Error getting properties for queue %s: %w", queueItem.Name, err)
```

Comment:
`%w` should be only used for creating error, not for printing. Use `%s` or `%v` instead.

https://go.dev/play/p/PJyrAD43KPr
![image](https://user-images.githubusercontent.com/17336997/220708762-8687c209-a6ba-4e6b-8774-40ccfcff65a2.png)

And after changing to `%s`:
![image](https://user-images.githubusercontent.com/17336997/220709083-6083be34-7096-4bcf-97ff-364946a24c36.png)